pinot-servicemanager/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+# uses 2.4 so we can use condition: service_healthy
```

Comment:
we can use docker compose file `.github/workflows/servicemanager/docker-compose.yml` here too.

Reply:
The files are slightly different - the github workflow assumes a local build tag (because the tagging is based off circleCI env variables). Long term, we don't want the duplication, but if we're duplicating the CI part we might as well duplicate the whole thing so we can remove one cleanly. That said, this file should be in the circleci dir - let me fix that. I guess while I'm in there I'll fix the GH action trigger too.